packages/components/src/__future__/Tag/Tag.module.scss

```diff
@@ -0,0 +1,79 @@
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/spacing";
+@import "~@kaizen/design-tokens/sass/typography";
+
+.container {
+  display: inline-flex;
+  align-items: center;
+  padding: 0.3rem $spacing-8 0.3rem $spacing-8;
+  gap: $spacing-4;
+  border-radius: 28px;
+  font-family: $typography-paragraph-intro-lede-font-family;
+  font-size: $typography-paragraph-small-font-size;
+  font-weight: $typography-paragraph-small-font-weight;
+  line-height: $typography-paragraph-small-line-height;
+  white-space: nowrap;
+}
+
+.iconContainer {
+  display: inline-flex;
+}
+
+.gray {
+  background-color: $color-gray-200;
```

Comment:
Spoke to @jmitsios and this was a mistake in the Figma. It has been fixed to be `$color-gray-300`, so this needs to be updated to match :)